New version: PartiallyObservedMarkovProcesses v0.4.12

JuliaRegistrator opened the pull request:
- Registering package: PartiallyObservedMarkovProcesses
- Repository: https://github.com/kingaa/PartiallyObservedMarkovProcesses.jl
- Created by: @kingaa
- Version: v0.4.12
- Commit: 461a02fbfecc0407c17e4ac13fbd2fa486815e1c
- Reviewed by: @kingaa
- Reference: https://github.com/kingaa/PartiallyObservedMarkovProcesses.jl/issues/17#issuecomment-3671776490
- Description: pomp in Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
It is now possible to pass options to the vectorfield integrator when `vectorfield` is invoked. All named arguments will be passed as options to the `DifferentialEquations.solve` call.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598f8842c05582000ec0f9deccf2156f24b52a358e6b62782a0b2273bb3e2f0ad0a9bfab7b95f9fa2aba9df3275cbf241b00cdf5fbb1ab58824ef7d64e7d91078785b722587ca6522304bacfd8bdcbfc84620758d039c9fc33e4124117895bfd657770b28cc77d41a59d00d0c0f481ab3a6ebe039e9a520fd1bb4f88ce1ad34eab3b2ccfd7ddab93ca7d1e2ae55d246784b -->

kingaa commented:
Stop this merge.

kingaa commented:
[noblock] I removed a comment that might have been blocking.  This will hopefully resolve the impasse.

JuliaRegistrator committed:
New version: PartiallyObservedMarkovProcesses v0.4.12

UUID: 32433216-4d85-4b2d-94ed-a614743ba12b
Repo: https://github.com/kingaa/PartiallyObservedMarkovProcesses.jl.git
Tree: 1f4a7465921703b269e6477ff78866e3c75694dd

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
